prisma/schema.prisma

```diff
@@ -161,20 +161,29 @@ model client_report {
   client                        String
   client_address                String?
   organization_name             String
+  application_url               String?
   storage_provider_distribution client_report_storage_provider_distribution[]
   replica_distribution          client_report_replica_distribution[]
   cid_sharing                   client_report_cid_sharing[]
   check_results                 client_report_check_result[]
 }
 
 model client_report_storage_provider_distribution {
-  id               BigInt                                                @id @default(autoincrement()) @db.BigInt
-  client_report    client_report                                         @relation(fields: [client_report_id], references: [id])
-  client_report_id BigInt                                                @db.BigInt
+  id               BigInt                                                      @id @default(autoincrement()) @db.BigInt
+  client_report    client_report                                               @relation(fields: [client_report_id], references: [id])
+  client_report_id BigInt                                                      @db.BigInt
   provider         String
   total_deal_size  BigInt
   unique_data_size BigInt
   location         client_report_storage_provider_distribution_location?
+  retrievability   client_report_storage_provider_distribution_retrievability?
```

Comment:
Maybe just store `Float` directly here? Seems like we have 1-to-1 relationship here and there's not that much data there to justify a separate table (as opposed to `location`).